db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/QueryTreeNode.java

```diff
@@ -957,6 +957,29 @@ public void init(Object arg1,
         }
     }
 
+    @Override
+    public void init(Object arg1,
+                     Object arg2,
+                     Object arg3,
+                     Object arg4,
+                     Object arg5,
+                     Object arg6,
+                     Object arg7,
+                     Object arg8,
+                     Object arg9,
+                     Object arg10,
+                     Object arg11,
+                     Object arg12,
+                     Object arg13,
+                     Object arg14,
+                     Object arg15,
+                     Object arg16,
+                     Object arg17) throws StandardException{
+        if(SanityManager.DEBUG){
+            SanityManager.THROWASSERT("Fourteen-argument init() not implemented for "+getClass().getName());
```

Comment:
Error message should indicate "Seventeen-argement init()" instead of "Fourteen-argument init()".